src/service/pokemonService.js

```diff
@@ -36,10 +40,38 @@ class Question {
         return Math.floor(Math.random() * numOfAnswers) + 1;
     }
 
-    // getNextQuestion() {
-    //     currentId = randomPokemonId();
+    getNextQuestion() {
+        const answersList = this.answers();
+        const correctAnswerId = () => answersList[this.correctAnswer];
+        answersList.map((id) => getPokemonById(id));
 
-    // }
+        
+        if (this.mode === 1) {
+            var currQuestion = {
+                pokemonUrl: getPokemonById(correctAnswerId).photoUrl
```

Comment:
Initially this object should have two values: { pokemonName: 'Bulbasaur', pokemonUrl: 'someUrl', } but if user chooses for instance mode=1, it won't be neccessary to have pokemonName in question (only pokemonUrl will be usefull).

Reply:
Schame for object which will be returned by method was tested in questionService.spec.js.